tests/Integration/tests/table-csv-table-with-markup/input/test.csv

```diff
@@ -0,0 +1,4 @@
+col 1, col 2
+**a**, "quoted column is unquoted"
```

Comment:
I am not sure this makes sense in most use cases... Often csv files would come from others sources so I would not really expect rst markup to be applied? What you think?

Reply:
According to this page: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#csv-files

it is supported by sphinx?

Reply:
Ok, didnt know that